recipes/_user.rb

```diff
@@ -40,7 +40,11 @@
   supports manage_home: true
   password node['omnibus']['build_user_password']
   unless windows?
-    shell '/bin/bash'
+    if mac_os_x? && node['platform_version'].satisfies?('>= 10.11')
+      shell '/usr/local/bin/bash' if mac_os_x? && node['platform_version'].satisfies?('>= 10.11')
+    else
+      shell '/bin/bash'
+    end
```

Comment:
We'll want all platforms to use a shell of `/usr/local/bin/bash` since this is the one we install.